spark/src/main/scala/org/apache/comet/CometExecIterator.scala

```diff
@@ -133,29 +133,25 @@ class CometExecIterator(
 
   def getNextBatch(): Option[ColumnarBatch] = {
     assert(partitionIndex >= 0 && partitionIndex < numParts)
+    try {
+      nativeLib.traceBegin("CometExecIterator_getNextBatch")
```

Comment:
should it be inside `if (memoryProfilingEnabled` ?

Reply:
I have now renamed `memoryProfilingEnabled` to `tracingEnabled`, and it is used consistently now